docs/scripts/generate_skills_docs.py

```
@@ -0,0 +1,150 @@
+    else:
+        logger.warning("Could not fetch marketplace.json")
+        skills = []
+
+    for skill in skills:
+        readme_url = f"{SKILLS_RAW_BASE}skills/{skill.slug}/README.md"
+        readme_content = _fetch(readme_url)
+        if readme_content:
+            page_content = _generate_skill_page(skill, readme_content)
+            page_path = skills_ecosystem_dir / f"{skill.slug}.md"
+            page_path.write_text(page_content, encoding="utf-8")
+            logger.info(f"Wrote {page_path}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)